packages/components/ovh-shell/README.md

```diff
@@ -1 +1,200 @@
+```
+
+By passing an app name to this method, you will have access to the shell plugins.
+
+_Example_:
+
+```ts
+import { useShellClient } from '@ovh-ux/shell';
+
+useShellClient('myAwesomeApp').then((shell: Shell) => {
+  // Do stuff with shell
+})
+
+// You can also do this : 
+
+const shell: Shell = await useShellClient('myAwesomeApp');
+
+// Do stuff with shell
+
+```
+
+## Client Side API
+
+### `environment`
+
+- `getEnvironment() => Environment`: returns current environment object.
+- `setUniverse(universe: string) => void`: Sets universe in the current Environnement.
+- `setApplication(applicationId: ApplicationId)`: Sets current application id
+
+_Example :_
+
+```ts
+// This example assumes you already have a shell defined
+
+const environmentPlugin = shell.getPlugin('environment');
+
+// Fetch locale
+const env: Environment = await environmentPlugin.getEnvironment();
+```
+
+### `i18n`
+
+- `getLocale`(`(shell: Shell, environment: Environment) => string`): Given a shell and an environment, returns the current locale of the user.
+- `setLocale`(`(locale: string) => void`): Sets new user locale based on given parameter.
+- `onLocaleChange`(`(callback: CallableFunction) => void`): defines a function to be triggered when the locale changes.
+
+_Example :_
+
+```ts
+// This example assumes you already have a shell defined
+
+const i18n = shell.getPlugin('i18n');
+
+// Fetch locale
+const oldLocale = await i18n.getLocale();
+
+// Set locale
+i18n.setLocale('en_GB');
+
+// On local change setup
+const dummyFunction = () => console.log('dummy');
+
+i18n.onLocaleChange(dummyFunction);
+
+// Each time the locale changes, your console will print out "dummy"
+```
+
+### `auth`
+
+- `login()`: Login to the manager.
+- `logout()`: Logout from the manager.
+
+_Example :_
+
+```ts
+const auth = shell.getPlugin('auth');
+
+auth.login();
+auth.logout();
+```
+
+### `ux`
+
+This plugin handles everything that is UI related, such as elements shown or hidden, the state or position of certain elements etc.
+Currently, it handles the state of the account sidebar, notifications sidebar and verifies cookie verification for the ssoAuthModal.
+
+- `showAccountSidebar() => void`) : display the account sidebar.
+- `disableAccountSidebarToggle() => void`) : Disallows toggle of the Account Sidebar. After calling this function, the Account Sidebar won't toggle anymore unless you call `enableAccountSidebarToggle`.
+- `enableAccountSidebarToggle() => void`) : Allows Account sidebar to be toggled.
+- `isAccountSidebarVisible() => boolean`) : Returns if Account Sidebar is shown or not.
+- `setForceAccountSiderBarDisplayOnLargeScreen(isForced: boolean) => void`) : forces the display on large screens. This option can be disabled by passing the false parameter to `isFalse`.
+- `resetAccountSidebar() => void` : Sets the Account Sidebar to its default state.
+- `isMenuSidebarVisible() => boolean` : Returns if the PNR menu is displayed.
```

Comment:
**question:** Shouldn't we found a name for this menu? PNR sounds more like a project name.